File: lab-3-ddb/environment/node_8.10.0/upload_items.js

var 
    AWS = require("aws-sdk"),                               
    DDB = new AWS.DynamoDB({
        apiVersion: "2012-08-10",
        region: "<FMI>"
    });                                                     

(function uploadItemstoDynamoDB(){
    var 
        cat_1 = {
            <FMI>{
                "petname":{
                    S: "Puddles"
                },
                "breed":{
                    S: "Russian Blue"
                }
            }, 
            <FMI>: "TOTAL", 
            <FMI>: "lostcats"
        };
     DDB.putItem(cat_1, function(err, data){   
         console.log(err, data);         
     });
     var 
        cat_2 = {
            <FMI>:{
                "petname":{
                    S: "Hosepipe"
                },
                "breed":{
                    S: "Scottish Fold"
                }
            }, 
            <FMI> "TOTAL", 
            <FMI>: "lostcats"
        };
     DDB.<FMI>(cat_2, function(err, data){   
         console.log(err, data);         
     });
})();

File: lab-3-ddb/environment/ruby_2.6.0/solution/upload_items.rb
require "aws-sdk-dynamodb"

dynamodb_client = Aws::DynamoDB::Client.new(region: "us-east-1")


resp = dynamodb_client.put_item({
  item: {
    "petname" => "Puddles", 
    "breed" => "Russian Blue"
  }, 
  return_consumed_capacity: "TOTAL", 
  table_name: "lostcats"
})

puts resp.to_h


resp = dynamodb_client.put_item({
  item: {
    "petname" => "Hosepipe", 
    "breed" => "Scottish Fold"
  }, 
  return_consumed_capacity: "TOTAL", 
  table_name: "lostcats" 
})

puts resp.to_h